src/components/MenuItemRenderHTMLTitle/index.native.js

```diff
@@ -7,7 +8,11 @@ const propTypes = menuItemRenderHTMLTitlePropTypes;
 const defaultProps = {};
 
 function MenuItemRenderHTMLTitle(props) {
-    return <RenderHTML html={props.title} />;
+    return (
+        <View>
+            <RenderHTML html={props.title} />
+        </View>
```

Comment:
I've just checked on web, android and ios (both native) that the above code, and the code we have in `index.js`:

```
<View style={styles.renderHTMLTitle}>
   <RenderHTML html={props.title} />
</View>
```

(only style is added here) gives exactly the same effect, so I think having separate file for native devices seems to be unnecessary.

Reply:
Okay updating 👍